src/freenet/node/PeerNode.java

```diff
@@ -337,10 +337,12 @@
 	protected long peerAddedTime = 1;
 	/** Average proportion of requests which are rejected or timed out */
 	private TimeDecayingRunningAverage pRejected;
-	/** Total low-level input bytes */
-	private long totalBytesIn;
-	/** Total low-level output bytes */
-	private long totalBytesOut;
+
+	/** Bytes received at/before startup */
+	private long bytesInAtStartup;
+	/** Bytes sent at/before startup */
+	private long bytesOutAtStartup;
```

Comment:
Good point, agreed. I probably won't update until ~tomorrow when I
hopefully will have revived my desktop.
On Mar 18, 2016 2:41 PM, "Bert Massop" <notifications@github.com> wrote:

> In src/freenet/node/PeerNode.java
> <https://github.com/freenet/fred/pull/517#discussion_r56726334>:
>
> > @@ -337,10 +337,12 @@
> >  	protected long peerAddedTime = 1;
> >  	/** Average proportion of requests which are rejected or timed out */
> >  	private TimeDecayingRunningAverage pRejected;
> > -	/** Total low-level input bytes */
> > -	private long totalBytesIn;
> > -	/** Total low-level output bytes */
> > -	private long totalBytesOut;
> > +
> > +	/** Bytes received at/before startup */
> > +	private long bytesInAtStartup;
> > +	/** Bytes sent at/before startup */
> > +	private long bytesOutAtStartup;
>
> Bytes at startup sounds like a fixed value. These should be final.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/freenet/fred/pull/517/files/691c6db3bcb61bbf8a89d835ed04321d97380c37#r56726334>
>